.trivyignore

```diff
@@ -0,0 +1,11 @@
+# WARNING - THIS FILE WAS GENERATED BY THE dps-gradle-spring-boot GRADLE PLUGIN
+ #           AND ANY MANUAL CHANGES WILL BE OVERRIDDEN ON YOUR NEXT BUILD.
+ #
+ # To make general changes to the suppressions below, change the gradle plugin dps-gradle-spring-boot,
+ # publish a new version and update to the new version in your gradle build script
+ #
+ # To stop the dps-gradle-spring-boot project from overwriting any project specific customisations here, remove the
+ # warning at the top of this file.
+
+ # suppress jackson warning until 2.14 is published
+ CVE-2020-36518
```

Comment:
I don't think this will pacify `hmpps/gradle_owasp_dependency_check` 💭

Reply:
Oh and also that the nightly scans work on the latest (dev) image, rather than the one in production ☘️